src/utils/index.ts

```diff
@@ -144,6 +150,6 @@ export function formatData(data: Data, args: Inputs) {
                 code: formatCode(script.code, scriptUrlParamInputs),
               };
         })
-      : null,
+      : undefined,
```

Comment:
I'm not sure how I feel about returning `undefined` instead of `null`. Is there a particular reason why switched it here?

Reply:
The `html` key is optional and therefore can potentially not exist anyways. 
This is just cleaner since the consumer needs to check if it exists either way.